ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -610,10 +612,23 @@ module.exports = function OpenlayersMap(
     showMultiLineShape: function(locationModel) {
       let lineObject = locationModel
         .get('multiline')
-        .map(line => line.map(coords => convertPointCoordinate(coords)))
+        .map(line => line.map(coords => unconvertPointCoordinate(coords)))
```

Comment:
The main thing I am unsure about in this PR is the conversions I do. My thought is that I need to `unconvert` the coordinates before performing Turf operations and then `convert` them before creating the openlayer geometries.